package.json

```diff
@@ -38,7 +38,7 @@
     "test": "touch src/theme/variables-custom.scss ; karma start",
     "test-api": "better-npm-run test-api",
     "test-mocha": "NODE_ENV=test ./node_modules/.bin/mocha api/test",
-    "integration": "integration-loader && integration all",
+    "integration": "integration-loader && (integration setup > /dev/null 2>&1) && integration test",
```

Comment:
I find some of the output of `integration setup` quite useful (e.g. to see which versions were installed) and, hence, I would not redirect all output to /dev/null.

If you want to make the build output less chatty I propose to change the `build` command.